pycalphad/core/composition_set.pxd

```diff
@@ -9,4 +9,8 @@ cdef public class CompositionSet(object)[type CompositionSetType, object Composi
     cdef public bint fixed
     cdef readonly double energy
     cdef double[::1] _energy_2d_view
+    cdef FastFunction phase_local_cons_func
+    cdef FastFunction phase_local_cons_jac
```

Comment:
Are these actually called or used anywhere? I didn't find anywhere that they are accessed.

Reply:
Yes, see my [related comment](https://github.com/pycalphad/pycalphad/pull/432#discussion_r1676952312)